Rover/sailboat.h

```diff
@@ -116,4 +135,18 @@ class Sailboat
     uint32_t tack_clear_ms;         // system time when tack was cleared
     bool tack_assist;               // true if we should use some throttle to assist tack
     UseMotor motor_state;           // current state of motor output
+    // in irons variables ---
+    static constexpr int rotate_clockwise = -1;
+    static constexpr int rotate_anticlockwise = 1;
```

Comment:
I don't think we need helpers for these.

Reply:
removed